plugins/SLURM-Integration/install.sh

```diff
@@ -64,17 +64,26 @@ function update_slurm_conf()
     fi
 }
 
-function update_python_path_for_ufm_prolog_epilog()
-{
-    python_path=$(which python3)
-    sudo sed -i -e "s#python3_path#$python_path#g" $SLURM_DIR/$UFM_PROLOG_FILE
+function set_python3_path()
+{   local full_python3_path=$1
+    local file_name=$2
+    sed -i -e "s#python3_path#$full_python3_path#g" $SLURM_DIR/$file_name
     if [[ $? -ne 0 ]];then
-        echo "Warning:: Failed to replace the output of 'which pythn3' command with python3_path keyword in $SLURM_DIR/$UFM_PROLOG_FILE file, please try to do it manually."
+        echo "Failed to set PYTHONPATH for $file_name, please set it manually at $SLURM_DIR/$file_name by setting python3_path parameter, prior to running the UFM-Slurm integration."
```

Comment:
exit with error?

Reply:
it will just print the error message to the user, I don't think we need to exit the installation because the user can do it manually in case it's failed to update the files, what do you think?

Reply:
we should always abort on error with clear err